management/management.go

```diff
@@ -227,7 +227,7 @@ func (m *Management) request(method, uri string, v interface{}) error {
 		return newError(res.Body)
 	}
 
-	if res.StatusCode != http.StatusNoContent {
+	if res.StatusCode != http.StatusNoContent && res.StatusCode != http.StatusAccepted {
```

Comment:
Could you give some context over this check? What caused a response of status accepted? Just curious..

Reply:
The tests were failing on the deletion step, and according to https://auth0.com/docs/api/management/v2#!/Connections/delete_connections_by_id it is expected that `DELETE` can respond with 202 instead of 204
I assume that would better come as a separate PR, but without this change I was unable to get tests green for https://github.com/alexkappa/terraform-provider-auth0/pull/244

Reply:
Wow, I missed that! Thanks! And no worries, we can keep it in with this PR.